### ISSUE TITLE
Support for multiple charts in sub folders 

### ISSUE THREAD

**Issue by maireanu** [nflaig/semantic-release-helm#34]
It is possible to add support to update the version for all the sub folders to the same version for all the charts

├── charts #Contains chart file for Helm and default values for each cluster
│   ├── cluster-autoscaler
│   │   ├── Chart.yaml
│   │   └── values.yaml
│   ├── ingress-nginx
│   │   ├── Chart.yaml
│   │   └── values.yaml
│   ├── kubernetes-dashboard
│   │   ├── Chart.yaml
│   │   └── values.yaml
│   └── metrics-server
│       ├── Chart.yaml
│       └── values.yaml

**Comment by nflaig** [comment]
No, that's not possible at the moment, you can only update the `Chart.yaml` in a single folder as configured via `chartPath`.

There was a related issue https://github.com/nflaig/semantic-release-helm/issues/33 but it was about versioning all of the sub charts separately.

I don't think it's the responsibility of this package to do this, please check out https://github.com/pmowrer/semantic-release-monorepo or similar solutions.

If there is no existing solution to this could think about adding a flag to do this (ie. to recursively update all charts).

**Comment by maireanu** [comment]
I was thinking to a similar think , but it does need to define each folder in the config file : https://github.com/pmowrer/semantic-release-monorepo.

It will be nice if it does that dynamically and checks for the Chart.yaml in the sub folders

**Comment by nflaig** [comment]
Feel free to add this functionality, happy to review the changes. But note, we should avoid adding features to this package that can be resolved by combining it with other semantic-release packages.

**Comment by yohanb** [comment]
I have everything setup with [semantic-release-monorepo](https://github.com/pmowrer/semantic-release-monorepo) but the only thing missing is being able to pass the chart path dynamically.
If `chartPath` or `CHART_PATH` isn't defined, why not fallback to using the context's current working directory (cwd)?  That would fix this use case and is an easy change.

**Comment by nflaig** [comment]
> If chartPath or CHART_PATH isn't defined, why not fallback to using the context's current working directory (cwd)? 

seems reasonable, but it's it not possible to pass that in externally? otherwise feel free to open a PR to change default behavior, should be non breaking as we currently require it to be passed so anyone using the package would override the default anyways

**Comment by nflaig** [comment]
should be resolved by #35, maybe @yohanb you can share how you use it in your project?

**Comment by yohanb** [comment]
> should be resolved by [#35](https://github.com/nflaig/semantic-release-helm/pull/35), maybe [@yohanb](https://github.com/yohanb) you can share how you use it in your project?

Sure!
I have a mono-repo with `./charts/*` chart folders.  The semantic-release config extends `semantic-release-monorepo` and I run semantic-release through an npm script with lerna.

With the latest fix #35  (version `2.10`), semantic-release with monorepo sends the current working directory of the chart (ex: `./charts/my-awesome-chart`) to the plugin and is able to package/publish.

Here's a simplified setup (didn't test it as is):

`package.json`
```json
{
  "name": "foobar",
  "version": "1.0.0",
  "workspaces": [
    "charts/*"
  ],
  "devDependencies": {
    "lerna": "^8.2.1",
    "semantic-release": "^24.2.3",
    "semantic-release-helm3": "^2.10.0",
    "semantic-release-monorepo": "^8.0.2"
  },
  "scripts": {
    "semantic-release": "lerna exec --concurrency 1 -- semantic-release"
  },
  "author": "Yohan Belval <yohan.belval@gmail.com>",
  "license": "MIT"
}
```

`release.config.js`
```javascript
module.exports = {
  branches: ["main"],
  extends: "semantic-release-monorepo",
  plugins: ["semantic-release-helm3"],
}
```
